### ISSUE TITLE
[Suggestion]: Make Rules show up longer/permanently when initializing/logging into world before typing

### ISSUE THREAD

**Issue by Whyareuhere** [CreeperHost/CreeperHostGui#676]
### Is your feature request related to a problem? Please describe.

A problem with the implementation of MineTogether is that the rules, which do show up once the MineTogether chat has been toggled to be displayed, is that the rules get washed away by the chat really quickly. As in, I can't even read the first sentence quick.

Since MineTogether has such strict rules on expletives and being English-only it is critical for users to read this before chatting, as one slip up and they might not be able to use it forever

### Describe the solution you'd like

A system should be implemented where upon toggling the chat open (ex: no chatting after 5-10 minutes) the user should have the rules pop up and to type a quick word (ex: "accept") that confirms that they have once again read the rules and so forth.

### Describe alternatives you've considered

Simply displays the rules only (no chat) before typing in a quick word (ex: "accept") that allows then to view and chat.

### Additional context

I accidentally said a mild expletive and got banned. This is NOT a ban appeal, but rather a way to allow users to adhere better to the rules without having to remember or forget the rules, especially in times where there is an extremely high chat flow.

**Comment by JustJinxed** [comment]
Another issue is these rules spam chat every time you're inactive for more than 10 minutes. Maybe a better solution would be to forward the rules to a popup with accept button, rather than depend on a system notification spam?

**Comment by Aberzhulan** [comment]
I saw this was discussed in #377, but I'd like to add something that it doesn't seem many people have noticed. A side effect of the rules no longer being on the screen by the time someone decides to click over to the MT chat and speak. I've noticed a lot of first time chatters will speak in a non-English language, and then immediately get jumped on by everyone.

I realize moderating chat in a single language is difficult enough, but I believe requiring an active "accept" step to chat would solve the "English only" rule violations OR cancelling that first message and displaying the rules. As it stands now, it appears the only way to view the rules is to speak, and it will send that first message and pop the rules at the same time.
